processor/src/decorators/mod.rs

```diff
@@ -52,8 +52,9 @@ where
     // INJECTOR HELPERS
     // --------------------------------------------------------------------------------------------
 
-    /// Injects a node of the Merkle tree specified by the values on the stack at the head of the
-    /// advice tape. The stack is expected to be arranged as follows (from the top):
+    /// Injects a node of the Merkle tree specified by the values on the operand stack at the top
+    /// of the advice stack. The operand stack is expected to be arranged as follows (from the
+    /// top):
```

Comment:
Let's change the first sentence to: "Pushes a node of the Merkle tree specified by the word on the top of the operand stack onto the advice stack."

Reply:
Done!